src/client/activity/UserActivity.js

```diff
@@ -91,7 +74,7 @@ class UserActivity extends React.Component {
     authenticatedUserName: '',
   };
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
componentWillMount is not a place to load async data. componentWillMount triggers on SSR so it wastes resources.

Reply:
oh yeah good call will change